src/tools/pkcs15-init.c

```diff
@@ -2004,7 +2029,8 @@ do_read_pkcs12_private_key(const char *filename, const char *passphrase,
 		return SC_ERROR_CANNOT_LOAD_KEY;
 	}
 
-	CRYPTO_add(&user_key->references, 1, CRYPTO_LOCK_EVP_PKEY);
+	EVP_PKEY_up_ref(user_key);
```

Comment:
Crypto_lock is not supported in 1.1.0 See:
https://github.com/openssl/openssl/issues/1260

If we need this, we may have to rewrite other code too.